src/Azure/Azure.Quantum.Client/SubmitterFactory.cs

```diff
@@ -127,7 +127,7 @@ public static class SubmitterFactory
             }
 
             (var constructorName, var constructorType) = constructorInfo.Value;
-            var args = new object?[] { target, workspace, storageConnection };
+            var args = new object?[] { target, workspace, targetCapability, storageConnection };
```

Comment:
It looks like these are in the wrong order... above, the `targetCapabilitiy` is added to the end of the argument list. This ordering would pass `targetCapability` as `storageConnection` and vice versa.

Reply:
Oh, never mind. This follows the ordering of a different API from another repo, not the ones above. That's a little confusion/tricky, since they are both strings and can easily be interchanged.